src/offset/fixed.rs

```diff
@@ -15,16 +16,85 @@ use crate::oldtime::Duration as OldDuration;
 use crate::DateTime;
 use crate::Timelike;
 
-/// The time zone with fixed offset, from UTC-23:59:59 to UTC+23:59:59.
+const OFFSET_NORMAL: i32 = 1;
+const OFFSET_UNKNOWN: i32 = 2;
+
+/// A fixed offset which can be used to indicate a time zone.
```

Comment:
Suggested opening sentence.

    /// A fixed offset of clock time from [Coordinated Universal Time] (UTC) stored to seconds granularity.
    /// A `FixedOffset` can be used to indicate a time zone.
    ///
    /// [Coordinated Universal Time]: https://en.wikipedia.org/wiki/Coordinated_Universal_Time

Reply:
Good suggestion.